docs/supervisor.md

```diff
@@ -11,15 +11,15 @@
 
 ## Compatibility
 
-| Velero Plugin for vSphere Version | vSphere Version    | Kubernetes Version                                                | vSphere CSI Driver Version | Velero Version | Deprecated | EOL Date      |
-|-----------------------------------|--------------------|-------------------------------------------------------------------|----------------------------|----------------|------------|---------------|
-| 1.4.0                             | 7.0U1c/P02 - 7.0U3 | Bundled with vSphere (1.16-1.19, 1.18-1.20, 1.19-1.22)            | Bundled with vSphere       | 1.8.1          | No         | N/A
-| 1.3.1                             | 7.0U1c/P02 - 7.0U3 | Bundled with vSphere (1.16-1.19, 1.18-1.20, 1.19-1.21)            | Bundled with vSphere       | 1.5.1          | No         | N/A           |
-| 1.3.0                             | 7.0U1c/P02 - 7.0U3 | Bundled with vSphere (1.16-1.19, 1.18-1.20, 1.19-1.21)            | Bundled with vSphere       | 1.5.1          | Yes        | December 2022 |
-| 1.2.1                             | 7.0U1c/P02 - 7.0U2 | Bundled with vSphere (1.16-1.19, 1.18-1.20)                       | Bundled with vSphere       | 1.5.1          | Yes        | June 2023     |
-| 1.2.0                             | 7.0U1c/P02 - 7.0U2 | Bundled with vSphere (1.16-1.19, 1.18-1.20)                       | Bundled with vSphere       | 1.5.1          | Yes        | December 2022 |
-| 1.1.1                             | 7.0U1c/P02         | Bundled with vSphere (1.16-1.19)                                  | Bundled with vSphere       | 1.5.1          | No         | N/A           |
-| 1.1.0                             | 7.0U1c/P02         | Bundled with vSphere (1.16-1.19)                                  | Bundled with vSphere       | 1.5.1          | Yes        | December 2022 |
+| Velero Plugin for vSphere Version | vSphere Version     | Kubernetes Version                                                | vSphere CSI Driver Version | Velero Version | Velero vSphere Operator Version | Deprecated | EOL Date      |
+|-----------------------------------|---------------------|-------------------------------------------------------------------|----------------------------|----------------|---------------------------------|------------|---------------|
+| 1.4.0                             | 7.0U3e/f/h          | Bundled with vSphere (1.22)                                       | Bundled with vSphere       | 1.8.1          | 1.2.0                           | No         | N/A           |
```

Comment:
`Bundled with vSphere (1.22) `
@swgupta-vmw Does this look good to you?  7.0U3e/f/h actually supports 1.20-1.22, but Velero vSphere Operator 1.2.0 only works with 1.22.  Should we add a note here?